src/components/CommentCard.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {ClockSVG} from './svgs/ClockSVG';
+import {HeartSVG} from './svgs/HeartSVG';
+
+export const CommentCard = ({ authorName, date, time, numHearts, content }) => {
+    return (
+        <div className='component-to-copy gb-comment-card'>
```

Comment:
the class component-to-copy was used just for the script that copied the elements, so you can take it down in here, is it doing nothing :D , forgot to tell you